plugins/com.google.cloud.tools.eclipse.googleapis/src/com/google/cloud/tools/eclipse/googleapis/internal/GoogleApi.java

```diff
@@ -33,52 +34,26 @@
  * </ol>
  */
 public enum GoogleApi {
-  APPENGINE_ADMIN_API(Messages.getString("api.appengine"), Appengine.DEFAULT_BASE_URL, //$NON-NLS-1$
-      "appengine.googleapis.com"), //$NON-NLS-1$
-  DATAFLOW_API(Messages.getString("api.dataflow"), Dataflow.DEFAULT_BASE_URL, //$NON-NLS-1$
-      "dataflow.googleapis.com"), //$NON-NLS-1$
-  CLOUDRESOURCE_MANAGER_API(Messages.getString("api.cloudresourcemanager"), //$NON-NLS-1$
-      CloudResourceManager.DEFAULT_BASE_URL, "cloudresourcemanager.googleapis.com"), //$NON-NLS-1$
-  CLOUD_STORAGE_API(Messages.getString("api.storage-json"), Storage.DEFAULT_BASE_URL, //$NON-NLS-1$
-      "storage-api.googleapis.com"), //$NON-NLS-1$
-  SERVICE_MANAGEMENT_API(Messages.getString("api.servicemanagement"), //$NON-NLS-1$
-      ServiceManagement.DEFAULT_BASE_URL, "servicemanagement.googleapis.com"); //$NON-NLS-1$
+  APPENGINE_ADMIN_API(Appengine.DEFAULT_BASE_URL, "appengine.googleapis.com"),
```

Comment:
Why did you toss the `$NON-NLS-X$`?

Reply:
Personally, I don't think every file has to have the comments. We can put it when we externalize strings for the first time for a file.